### ISSUE TITLE
Device Info page error

### ISSUE THREAD

**Issue by mrpau-richard** [learningequality/kolibri#2802]
### Observed behavior

After I created a Kolibri admin account and navigate at the device info page, I got this device Info page error.

### Errors and logs

Relevant errors and tracebacks from:

![screen shot 2017-12-04 at 2 08 43 pm](https://user-images.githubusercontent.com/8663934/33538903-b87f331a-d8fe-11e7-85f7-f82eeca3d229.png)

server logs
```
Installed 2 object(s) from 1 fixture(s)
[37mINFO     Started new MESSAGEPROCESSOR thread ID#kilo-bluebird-bluebird-cold[0m
[37mINFO     Started new SCHEDULER thread ID#two-nitrogen-lake-jupiter[0m
[37mINFO     Started new WORKERMESSAGEHANDLER thread ID#quiet-uncle-april-charlie[0m

```

### Steps to reproduce

Install the [pre-release](https://github.com/learningequality/kolibri/releases/tag/v0.7.0-beta5) of kolibri without any existing kolibri installation. 
After the installation and successfully ran the Kolibri server. Create a Kolibri admin account then, navigate at the device info page.

This issue can't reproduce when running a Kolibri server in `kolibri --debug manage devserver` command.

### Context

Kolibri version: v0.7.0-beta5
Operating system: Windows 7 32 bit & OS X Yosemite 10.10.5
Browser: Google chrome Version 62.0.3202.94 (Official Build) (32-bit) & Safari 8.0.8

**Comment by christianmemije** [comment]
Duplicate of #2713, should have been fixed in #2737

**Comment by indirectlylit** [comment]
hmm actually looks like @mrpau-dev is testing beta5, so  41eb556c65550c17b7aa14bf2fda03f94164ba85 should be in there

**Comment by rtibbles** [comment]
This seems to be an error with the device info endpoint, no? This seems more like a duplicate of https://github.com/learningequality/kolibri/issues/2801#issuecomment-349064951

**Comment by ralphiee22** [comment]
Will be resolved in morango-0.2.0.

**Comment by indirectlylit** [comment]
should have been resolved in #2496
